.github/workflows/push-tests.yml

```diff
@@ -74,12 +75,20 @@ jobs:
         with:
           registry: ${{ secrets.ECR_REPO }}
       - uses: actions/checkout@v2
-      - run: |
+      - name: Install dependencies
+        run: |
           npm ci
           npx playwright install --with-deps
           cp src/configs/dev.js src/configs/configs.js
           npm run dev&
-      - run: DEBUG=pw:api npm run e2e
+      - name: Run E2E tests
+        run: DEBUG=pw:api npm run e2e
+      - uses: actions/upload-artifact@v2
```

Comment:
uploads failure screenshots and videos, so we can download the zip under the workflow SUMMARY page "artifact" section AFTER the whole workflow completes